src/service/request_service.rs

```diff
@@ -128,7 +128,9 @@ mod tests {
             .unwrap();
         let mut builder = RequestServiceBuilder::new(router).unwrap();
         let mut service = builder.build(remote_addr);
-        poll_fn(|_| -> Poll<Result<(), Error>> { Poll::Ready(Ok(())) });
+        if let Err(e) = poll_fn(|ctx| -> Poll<Result<(), Error>> { service.poll_ready(ctx) }).await {
+            panic!("request service is not ready: {}", e);
+        }
```

Comment:
You can simply do `.await.expect("request service is not ready")`

Reply:
Fixed!